Assets/Labrys/Generation/Section.cs

```diff
@@ -1,34 +1,32 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using System.Collections.Generic;
 
-namespace Labrys.Generation
+namespace Labrys.Generation
 {
     /// <summary>
     /// A struct representing a single grid element during generation. This will
     /// map 1:1 to a Tile when generation is completed.
-    /// </summary>
-    [System.Serializable]
-    public struct Section
-    {
-        public Connection internalConnections;
-        public Connection externalConnections;
-        //public string variant;
-        private short variantID;
+    /// </summary>
+    [System.Serializable]
+    public struct Section
+    {
+        public Connection internalConnections;
+        public Connection externalConnections;
+        private readonly short variantID;
+        private Dictionary<string, string> metadata;

```

Comment:
this is *real* bad for efficiency; Section was designed to be a struct small enough to fit into a single int size so that the garbage collector wasn't overwhelmed. definitely want to see if there's a better way to implement this.

Reply:
Agreed. I thought about alternatives, but they all required pretty big structure changes, or were equally disgusting.  We should probably have a more in-depth discussion on this.

Reply:
Per discussion, these changes will be removed in favor of a flyweight pattern to be implemented after FeatureLibrary is done.